deeppavlov/utils/server/server.py

```diff
@@ -154,9 +171,12 @@ def index():
         }
     }
 
-    @app.route(model_endpoint, methods=['POST'])
+    @app.route(model_endpoint, methods=['POST', 'GET'])
```

Comment:
This way the `GET` option appears in Flasgger, it will confuse users. If you're doing `if ... else...` there, you can make a new endpoint without swagger, I feel.